seed.js

```diff
@@ -43,6 +47,93 @@ var seedUsers = function () {
 
 };
 
+let seedSongs = function(){
+  let songs = [
+    {
+      title: "Symphony No. 5 in C Minor",
+      subtitle: "First Movement",
+      description: "Adapted By Arrangement from Ernst Pauer, 1826-1905",
+      yearComposed: 1823,
+      fileName: "Beethoven_Symphony_No._5_1st_movement_Piano_solo.pdf",
+      price: 10.00,
+      inventoryQuantity: 10,
+      imageURL: "???",
+      instrumentTags: ["piano"],
+      sourceURL: "???",
+      publicDomainStatus: "public"
+    }
+  ]
+  let creatingSongs = songs.map(function (songObj) {
+      return Song.create(songObj);
+  });
+
+  return Promise.all(creatingSongs);
+};
+
+let seedComposers = function(seedSongs){
+  //populate this object with composer data. set an object title so that it can be used in the "setComposer" command below.
+  let composers = {
+    beethoven: {
+      firstName: "Ludwig",
+      lastName: "Van Beethoven"
+    }
+  }
+
+  // set composer for each song.
+  return seedSongs.spread( (symphonyNo5) => Promise.all([
```

Comment:
I think that's just so we can add more songs to the seed file later without changing the code. Is that right @thejohnbackes ?